cmd/gaia/app/export.go

```diff
@@ -117,7 +117,9 @@ func (app *GaiaApp) prepForZeroHeightGenesis(ctx sdk.Context, jailWhiteList []st
 
 	// reinitialize all delegations
 	for _, del := range dels {
+		app.distrKeeper.DeleteDelegatorStartingInfo(ctx, del.ValidatorAddress, del.DelegatorAddress)
```

Comment:
Do we test this somehow? Like a cli test for instance?

Reply:
Do we actually need this line? I think `initializeDelegation`, called in `AfterDelegationModified`, will overwrite any existing starting info.

It's OK with me to leave it for clarity though.

Reply:
nah - it's not required I'm deleting it. Thanks Chris

Reply:
Yes @alessio -- the invariants/simulation test this.

Reply:
@alessio yes this line _is_ tested in now that the simulation after import tests nolonger fail